src/csscolorparser/csscolorparser.cpp

```diff
@@ -110,9 +110,6 @@ const NamedColor namedColors[] = {
     { "yellow", { 255, 255, 0, 1 } }, { "yellowgreen", { 154, 205, 50, 1 } }
 };
 
-const size_t namedColorCount = sizeof (namedColors) / sizeof (NamedColor);
```

Comment:
Commited to upstream in https://github.com/mapbox/css-color-parser-cpp/commit/4f51c64fbce0eae1b24ca9ddb252eeba04f548f3